### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by MagicBlue256** [spghettywafer/InstagramMock-v2#2]
https://github.com/spghettywafer/InstagramMock-v2/blob/c9a67b66614e5437019dcca78f93de2690ef998d/controller/post-controller.go#L63-L68

What will happen if there are 2 `Create` with the same `file.Filename`?

**Comment by spghettywafer** [comment]
ohh hmm.. the second file will overwrite the first file (?)

So, is it better to save the file with a unique name, in this case, it should use 'postID'?

**Comment by MagicBlue256** [comment]
yes, you can use `postID`, but the problem is, at this point, do you have `postID` here?
